docs/sources/developers/plugins/custom-panel-option-editors.md

```diff
@@ -39,9 +43,9 @@ export const plugin = new PanelPlugin<SimpleOptions>(SimplePanel).setPanelOption
 
 ## Add settings to your panel option editor
 
-If you're using your custom editor to configure multiple options, you might want to be able to customize it. Add settings to your editor by setting the second template variable of `StandardEditorProps` to an interface that contains the settings you want to be able to configure.
+If you're using your custom editor to configure multiple options, you might want to be able to customize it. To add settings to your editor, set the second template variable of `StandardEditorProps` to an interface that contains the settings you want to configure. Access the editor settings through the `item` prop. 
```

Comment:
I'm struggling to understand this section, and in particular the first sentence...

> If you're using your custom editor to configure multiple options, you might want to be able to customize it.

What do we mean by multiple options?

Reply:
I mean multiple settings. Have re-written this. If you find this unclear, let me know now or in phase 3.